src/components/basic_table/in_memory_table.js

```diff
@@ -352,12 +361,12 @@ export class EuiInMemoryTable extends Component {
     // Data loaded from a server can have a default sort order which is meaningful to the
     // user, but can't be reproduced with client-side sort logic. So we allow the table to display
     // rows in the order in which they're initially loaded by providing an undefined sorting prop.
-    // Once a user sorts a column, this will become a fully-defined sorting prop.
     const sorting = !hasSorting ? undefined : {
       sort: (!sortField && !sortDirection) ? undefined : {
         field: sortField,
         direction: sortDirection,
       },
+      allowNeutralSort: true,
```

Comment:
I would set this to pass down the prop from line 75
```suggestion
      allowNeutralSort: allowNeutralSort,
```

and then set the prop to `true` as default after line 145 https://github.com/elastic/eui/blob/f02d7d804f0c80c15cda9fad97203b70845320b9/src/components/basic_table/in_memory_table.js#L145
`allowNeutralSort: true`

Reply:
This way consumers can override the default by setting it to false.